Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
This is what ultimately is in progress.

See, I could theoretically do it your way and make the entire RSP compile out of the box on Unix-based systems, in one big PR, so that no blockage can hide Windows compile errors so that no Windows compile errors would be happening.  The problem with this approach, is the PRs become enormous, full of changes all over the files for fixing WIN32-ism's.  So I find it wiser to make smart incremental, smaller, easy-to-review-and-criticize pull requests.